Multiprotocol/ExpressLRS.ino

```diff
@@ -0,0 +1,540 @@
+#if defined(ELRS_SX1276_INO) && defined(SX1276_INSTALLED)
+
+#include "iface_sx1276.h"
+#include "ExpressLRS.h"
+
+uint8_t UID[6] = {10, 183, 12, 124, 56, 3}; // default UID for ExpressLRS (for testing)
```

Comment:
this is just `default`, not `default ExpressLRS binding phrase` as in https://github.com/AlessandroAU/ExpressLRS/blob/dc00b8ca9db7e67516de3469f2528d08ce36ea98/src/user_defines.txt#L13

made a script to get the `uid`:

```
binding phrase: 'default ExpressLRS binding phrase'
     uid bytes: '165,221,171,247,138,255'

Multiprotocol/ExpressLRS.ino:
  uint8_t UID[6] = {165,221,171,247,138,255};
```

```python
#!/usr/bin/env python3

import argparse
import hashlib
import sys

parser = argparse.ArgumentParser(prog='PROG')
parser.add_argument('--phrase', type=str, default='default ExpressLRS binding phrase', help='binding phrase')
args = parser.parse_args()

bindingPhraseHash = hashlib.md5(args.phrase.encode()).digest()
UIDbytes = (str(bindingPhraseHash[0]) + "," + str(bindingPhraseHash[1]) + "," + str(bindingPhraseHash[2]) + ","+ str(bindingPhraseHash[3]) + "," + str(bindingPhraseHash[4]) + "," + str(bindingPhraseHash[5]))

sys.stdout.write("binding phrase: '%s'\n     uid bytes: '%s'\n\n" % ( args.phrase, UIDbytes))
sys.stdout.write("Multiprotocol/ExpressLRS.ino:\n  uint8_t UID[6] = {%s};\n" % UIDbytes)
sys.stdout.flush()
```

Reply:
Thanks dude! will update UID. Need to sort out eeprom then static UID can be done away with.